app/src/components/map/MapFeatureList.tsx

```diff
@@ -3,24 +3,34 @@ import Checkbox from '@mui/material/Checkbox';
 import FormGroup from '@mui/material/FormGroup';
 import FormControlLabel from '@mui/material/FormControlLabel';
 import React from 'react';
+import IconButton from '@mui/material/IconButton';
+import DeleteForeverOutlinedIcon from '@mui/icons-material/DeleteForeverOutlined';
+import Box from '@mui/material/Box';
+import get from 'lodash-es/get';
+import { recalculateFeatureIds } from 'utils/mapBoundaryUploadHelpers';
 
 export interface MapFeatureListProps {
   features?: any;
```

Comment:
Also good practice to Interface everything, helps a lot tracking bugs and configuring components

Reply:
Good point here.

Reply:
Think I got this covered.